sound/soc/sof/ipc4-pcm.c

```diff
@@ -79,21 +80,27 @@ sof_ipc4_add_pipeline_to_trigger_list(struct snd_sof_dev *sdev, int state,
 		 * Trigger pipeline if all PCMs containing it are paused or if it is RUNNING
 		 * for the first time
 		 */
-		if (spipe->started_count == spipe->paused_count)
+		if (spipe->started_count == spipe->paused_count) {
+			pipe_priority[trigger_list->count] = pipeline->priority;
```

Comment:
I think this sorting is not too effective at the moment:
https://github.com/thesofproject/linux/blob/topic/sof-dev/sound/soc/sof/ipc4-topology.c#L686-L687

Reply:
```c
	/* TODO: Get priority from topology */
	pipeline->priority = 0;
```

Reply:
sure, thanks !  I also find it when I set up my environment for debugging. Let's update it